terralab/version_utils.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import requests
+import os
+import json
+from datetime import datetime, date
+from importlib.metadata import version, PackageNotFoundError
+
+from terralab.config import load_config
+
+package_name = "terralab-cli"
```

Comment:
is it possible to get this from like the pyproject.toml file possibly or some poetry metadata something something?